### ISSUE TITLE
Feature request to support apcupsd by agent

### ISSUE THREAD

**Issue by mapietru** [bashclub/check-opnsense#26]
Might be intressting for those who have an ups from APC (Schneider). There is a plugin for checkmk with some nice output. 
https://exchange.checkmk.com/p/apcaccess

On opnsense side you just need to install the plugin _os-apcupsd_. 

For checkmk there is some mkp to parse the information from the client. https://exchange.checkmk.com/packages/apcaccess/914/apcaccess-5.0.mkp

the only thing missing is a small change in opnsense_checkmk_agent.py

I am not a coding guy, but this is my current solution:
``` 
def check_apcupsd(self):
        _ret = ["<<<apcaccess:sep(58)>>>"]
        _ret.append("[[apcupsd.conf]]")
        _ret.append(self._run_prog("apcaccess").strip())
        return _ret
```
The output of the checkmk service looks like this:

![image](https://user-images.githubusercontent.com/87319110/212846137-4b7b218b-b930-4ae2-8a4d-c896a64a9987.png)

![image](https://user-images.githubusercontent.com/87319110/212846732-a420b137-9285-40cf-8d1f-3ce70cbafdae.png)

**Comment by thorstenspille** [comment]
Please check testing branch, should be fixed with https://github.com/bashclub/check-opnsense/commit/e1c5d1a121059f61eaab5db6954f778e59b41693

**Comment by mapietru** [comment]
> Please check testing branch, should be fixed with [e1c5d1a](https://github.com/bashclub/check-opnsense/commit/e1c5d1a121059f61eaab5db6954f778e59b41693)

Updated this morning, looks good!
